ioctl/newcmd/account/accountcreate.go

```diff
@@ -65,9 +63,9 @@ func NewAccountCreate(c ioctl.Client) *cobra.Command {
 				if err != nil {
 					return output.NewError(output.CryptoError, failToGenerateNewPrivateKey, err)
 				}
-				addr, err := address.FromBytes(private.PublicKey().Hash())
-				if err != nil {
-					return output.NewError(output.ConvertError, failToConvertPublicKeyIntoAddress, err)
+				addr := private.PublicKey().Address()
+				if addr == nil {
+					return output.NewError(output.ConvertError, failToConvertPublicKeyIntoAddress, nil)
```

Comment:
will the replacement of err with "nil" work?

Reply:
yes, if there's error, returned `addr` will be nil

Reply:
@dustinxie @CoderZhi I guess no more change is required at my end so I updated the branch with latest master. Please review this PR.

Reply:
> will the replacement of err with "nil" work?

return output.NewError(output.AddressError, failToConvertPublicKeyIntoAddress, nil)